addOns/network/src/main/java/org/zaproxy/addon/network/GlobalExclusionsOptions.java

```diff
@@ -188,9 +189,156 @@ protected void updateConfigsImpl(int fileVersion) {
                     setGlobalExclusions(DEFAULT_GLOBAL_EXCLUSIONS);
                     break;
                 }
+            case 1:
+                updateToVersion2();
         }
     }
 
+    private void updateToVersion2() {
+        List<GlobalExclusion> rulesBeingUpdated = getVersion1DefaultRules();
+        List<HierarchicalConfiguration> fields =
+                ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
+        List<String> oldValues = new ArrayList<>(rulesBeingUpdated.size());
+        rulesBeingUpdated.forEach(e -> oldValues.add(e.getValue()));
+        for (HierarchicalConfiguration sub : fields) {
+            try {
+                String value = sub.getString(GLOBAL_EXCLUSION_VALUE_KEY, "");
+                if (oldValues.contains(value)) {
+                    String name = sub.getString(GLOBAL_EXCLUSION_NAME_KEY, "");
+                    // Fix naming mistake
+                    if (name.contains("adx")) {
+                        name = name.replace("adx", "axd");
+                        sub.setProperty(GLOBAL_EXCLUSION_NAME_KEY, name);
+                    }
+                    // Extend image types pattern(s)
+                    if (value.contains("|bmp)")) {
+                        value = value.replace("|bmp)", "|bmp|svg|webp)");
+                        sub.setProperty(GLOBAL_EXCLUSION_VALUE_KEY, value);
+                    }
+                    // Extend audio/video types pattern(s)
+                    if (value.contains("|og[gav])")) {
+                        value = value.replace("|og[gav])", "|og[gav]|webm)");
+                        sub.setProperty(GLOBAL_EXCLUSION_VALUE_KEY, value);
```

Comment:
Same, it's being done always when making case insensitive.

Reply:
? If commented out it isn't appended. Value is coming from the existing config not the v2 config.

Reply:
Not following what's not being appended and the relation to v2 config. In line 273 we are setting the value always.

Reply:
If I comment out those setProperty calls the new unit test fails and shows that they weren't updated

Reply:
Done